app/javascript/gobierto_data/webapp/components/Dataset.vue

```diff
@@ -383,6 +397,44 @@ export default {
     this.$root.$off('showSavingDialogEvent')
   },
   methods: {
+    updateBaseTitle() {
+      this.$nextTick(() =>
```

Comment:
I've been always curious, why two nextTick?

Reply:
That's one of the reasons to start commenting the code. My bad here. I don't remember it pretty much.

I think it's related with the vue-router cycles. The `afterEach` hook is triggered BEFORE the DOM has been mounted, something like that. It could be confirmed using console logs in all cycles to check the actual order.

Reply:
@ferblape In this case is totally unnecesary.